src/main/java/com/autotune/utils/Utils.java

```diff
@@ -171,7 +171,7 @@ public static KruizeObject convertCreateExperimentSOToKruizeObject(CreateExperim
 						if (null != objectType)
 							deploymentObject.setType(objectType);
 						HashMap<String, ContainerObject> containerObjectHashMap = new HashMap<String, ContainerObject>();
-						for (ContainerObject containerObject: k8sObject.getContainers()) {
+						for (ContainerObject containerObject: k8sObject.getContainerObjects()) {
```

Comment:
Please move the entire `utils.Utils.Converters` class to `analyzer.serviceObjects.Converters`. This is used only in the analyzer and there is no reason this code has to be in common utils.

Reply:
Moved